src/attributes/components/AttributeList/AttributeList.tsx

```diff
@@ -83,23 +83,39 @@ const AttributeList: React.StatelessComponent<AttributeListProps> = ({
         toolbar={toolbar}
       >
         <TableCell className={classes.colSlug}>
-          {i18n.t("Attribute Code", { context: "attribute slug" })}
+          <FormattedMessage
+            defaultMessage="Attribute Code"
+            description="attribute list: slug column header"
+            id="attributeListSlugColumnHeader"
+          />
         </TableCell>
         <TableCell className={classes.colName}>
-          {i18n.t("Default Label", { context: "attribute name" })}
+          <FormattedMessage
+            defaultMessage="Default Label"
+            description="attribute list: name column header"
+            id="attributeListNameColumnHeader"
+          />
         </TableCell>
         <TableCell className={classes.colVisible}>
-          {i18n.t("Visible", { context: "attribute visibility" })}
+          <FormattedMessage
+            defaultMessage="Visible"
+            description="attribute list: visibility column header"
```

Comment:
Here we should specify only that the subject is a single attribute. It affects translation because in English it's the same for both genders, and singular/plural ("widoczny"/"widoczna"/"widoczne").